packages/combobox/src/ComboboxMenu/Menu.styles.ts

```diff
@@ -3,20 +3,29 @@ import { transparentize } from 'polished';
 import { css, keyframes } from '@leafygreen-ui/emotion';
 import { Theme } from '@leafygreen-ui/lib';
 import { palette } from '@leafygreen-ui/palette';
-import { fontFamilies, spacing, typeScales } from '@leafygreen-ui/tokens';
+import { fontFamilies, spacing } from '@leafygreen-ui/tokens';
 
+import { fontSize, lineHeight } from '../Chip/Chip.styles';
 import { ComboboxSize as Size } from '../Combobox.types';
 
 export const menuItemPadding: Record<Size, { x: number; y: number }> = {
+  [Size.XSmall]: { x: 12, y: 8 },
+  [Size.Small]: { x: 12, y: 8 },
   [Size.Default]: { x: 12, y: 8 },
   [Size.Large]: { x: 12, y: 8 },
 };
 
+/** Util that returns the height of a menu item (in px) */
+const getMenuItemHeight = (size: Size) => {
+  return lineHeight[size] + 2 * menuItemPadding[size].y;
+};
+
 /** Height of a menu item (in px) */
-export const menuItemHeight = {
-  [Size.Default]:
-    typeScales.body1.lineHeight + 2 * menuItemPadding[Size.Default].y,
-  [Size.Large]: typeScales.body2.lineHeight + 2 * menuItemPadding[Size.Large].y,
+export const menuItemHeight: Record<Size, number> = {
+  [Size.XSmall]: getMenuItemHeight(Size.XSmall),
+  [Size.Small]: getMenuItemHeight(Size.Small),
+  [Size.Default]: getMenuItemHeight(Size.Default),
+  [Size.Large]: getMenuItemHeight(Size.Large),
 };
```

Comment:
Why do we need an object to wrap a function call here? We could just call `getMenuItemHeight(Size.XSmall)` from whatever references this object